frontend/config/routes.rb

```diff
@@ -21,6 +21,8 @@
     resources :coupon_codes, only: :create
   end
 
+  post '/orders/content', to: 'order_contents#create'
```

Comment:
@kennyadsl Hi! I have been thinking about this route's placement as well trying to think how the routes will be after making the `OrdersController` RESTful and what would be best. I thought maybe it could be good to discuss this a little more in #4125